src/WebJobs.Extensions.Http/Config/HttpExtensionConfiguration.cs

```diff
@@ -70,6 +70,7 @@ public void Initialize(ExtensionConfigContext context)
             }
 
             context.Config.RegisterBindingExtension(new HttpTriggerAttributeBindingProvider(SetResponse));
+            context.Config.RegisterBindingExtension(new HttpDirectRequestBindingProvider());
```

Comment:
For some reason when this binding was ported to v2, this provider was completely removed